### PR TITLE
Create build folder if it doesn't exist when running `create-example-tests.js`

### PR DESCRIPTION
**Pull request by howard-e**
[Preview Tests](https://raw.githack.com/w3c/aria-at/bocoup/fix-issue-588/build/index.html)

Address #588.

### CODE REVIEW

File: scripts/create-example-tests.js [hunk 1]
```diff
@@ -97,6 +98,9 @@ const createExampleTests = async ({ directory, args = {} }) => {
   const testPlanBuildDirectory = path.join(buildDirectory, directory);
   const indexFileBuildOutputPath = path.join(testPlanBuildDirectory, 'index.html');
 
+  // create build directory if it doesn't exist
+  fs.existsSync(buildDirectory) || fs.mkdirSync(buildDirectory);
```
Comment (mzgoddard) [inline scripts/create-example-tests.js:102]: I think we can use the `recursive` option here and remove the `existsSync` call.

```js
fs.mkdirSync(buildDirectory, {recursive: true});
```

https://nodejs.org/dist/latest-v16.x/docs/api/fs.html#fsmkdirsyncpath-options

**Review by mzgoddard — CHANGES_REQUESTED** [review]
Looks good.

File: scripts/create-example-tests.js (hunk 1 above)
Comment (howard-e) [reply scripts/create-example-tests.js]: Pushed the recommended change @mzgoddard

**Review by mzgoddard — APPROVED** [review]
👍

**Comment by jscholes** [comment]
Merging this in the meantime, while https://github.com/w3c/aria-at/pull/621 is being worked on, because it's still causing us problems.

**Comment by jscholes** [comment]
Or maybe not, because there are tons of merge conflicts.

**Review by alflennik — APPROVED** [review]
Looks good @howard-e!

**Comment by howard-e** [comment]
> Or maybe not, because there are tons of merge conflicts.

I've gone ahead and resolved the conflicts and merged @jscholes

**Pull request merged**